docs/diagnostics/PX1085.md

```diff
@@ -0,0 +1,35 @@
+# PX1085
+This document describes the PX1085 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                     | Type                           | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------------------------------- | ------------------------------ | ----------- | 
+| PX1085 | BQL statements and other database queries should not be executed during the `PXGraph` initialization. | Warning (Level 1: Significant) | Unavailable |
```

Comment:
The diagnostic is executed only if ISV specific option is activated.

Reply:
This is written in the end of the description.